web_form_sticky_header/static/src/js/web_form_sticky_header.js

```diff
@@ -0,0 +1,24 @@
+//-*- coding: utf-8 -*-
```

Comment:
This is only for python files, not javascript

Reply:
@NL66278 Yes wrong vim, I will reimplement this module